### PR TITLE
convert subpar/ampvelocityscale to float

### PR DESCRIPTION
**Pull request by michiboo**
Hi I convert amp velocity scale function to float in subpar!

### CODE REVIEW

File: schema/test.json [hunk 1]
```diff
@@ -6753,13 +6753,13 @@
...
         "shortname": "sense",
-        "name"     : "PAmpVelocityScaleFunction",
+        "name"     : "AmpVelocityScaleFunction",
         "tooltip"  : "Amplitude Velocity Sensing function",
-        "type"     : "i",
-        "range"    : [0,127],
-        "default"  : "90"
+        "type"     : "f",
+        "range"    : [0,100.0],
+        "default"  : "50"
```
Comment (JohannesLorenz) [inline schema/test.json]: This doesn't match the default value from the zynaddsubfx commit. Is this difference wanted?

Comment (michiboo) [reply schema/test.json]: It was really silly for me to miss that 😅 thanks for the review!

**Comment by michiboo** [comment]
Hi, Changed as requested!

**Comment by fundamental** [comment]
Thanks for the improvement. The other two PRs have been merged, though @JohannesLorenz was talking about a discrepancy with this PR on IRC. Did you end up modifying the json schema by hand?

It can be generated from the zynaddsubfx with the --dump-json-schema flag.

**Comment by michiboo** [comment]
Hi I did changed it manually, but now I replaced it with the generated schema

**Comment by fundamental** [comment]
Thanks

**Pull request merged**